src/main/osd/osd_elements.c

```diff
@@ -578,13 +579,29 @@ static void osdElementEscTemperature(osdElementParms_t *element)
     }
 }
 
+#endif // USE_ESC_SENSOR
+
+#if defined(USE_ESC_SENSOR) || defined(USE_RPM_FILTER)
+int getEscRpmFreq(int i ) 
```

Comment:
Please remove extra spaces, and add spaces around operators, as stipulated by the coding style: https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#spaces

Reply:
should be fix now.